src/cascadia/TerminalSettingsEditor/ColorSchemes.h

```diff
@@ -5,27 +5,37 @@
 
 #include "ColorTableEntry.g.h"
 #include "ColorSchemes.g.h"
+#include "ColorSchemesPageNavigationState.g.h"
 #include "Utils.h"
 
 namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 {
+    struct ColorSchemesPageNavigationState : ColorSchemesPageNavigationStateT<ColorSchemesPageNavigationState>
+    {
+    public:
+        ColorSchemesPageNavigationState(Model::GlobalAppSettings settings) :
+            _Globals{ settings } {}
+
+        GETSET_PROPERTY(Model::GlobalAppSettings, Globals, nullptr);
+    };
+
     struct ColorSchemes : ColorSchemesT<ColorSchemes>
     {
         ColorSchemes();
 
+        void OnNavigatedTo(winrt::Windows::UI::Xaml::Navigation::NavigationEventArgs e);
```

Comment:
ideally these would be `const &` buckos, but that's an annoyingly large change to make across this PR ;P